packages/core/src/utils/utils.js

```diff
@@ -108,23 +109,49 @@ export const getContrastRatio = (colorA, colorB) => {
   return (Math.max(luminA, luminB) + 0.05) / (Math.min(luminA, luminB) + 0.05)
 }
 
+export const getBreakpointSize = (array, length) => {
+  const filteredArray = array.slice(0, length)
+  return filteredArray.reduceRight((carry, item) => {
+    return !carry ? item : carry
+  }, null)
```

Comment:
Can we add some comments describing the purpose of each of the steps in this function? It'll make it easier to maintain.

Reply:
Updated 👍 (open to adding even more detail if you think necessary)